storage/repository/uri.go

```diff
@@ -13,6 +15,9 @@ import (
 // Declare conformance with fyne.URI interface.
 var _ fyne.URI = &uri{}
 
+// Regex pattern to match Windows drive paths
```

Comment:
This probably needs a note about them being URI related or the slash in the middle may be unobvious or look like duplication of the other prefix pattern